laravel/database/migrations/2021_04_09_145328_drop_attending_column_from_option.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DropAttendingColumnFromOption extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('option', function (Blueprint $table) {
+            $table->dropColumn("attending_id");
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('option', function (Blueprint $table) {
+            $table->Integer("attending_id");
```

Comment:
correct me if I'm wrong, but I believe it's supposed to be lowercase `integer`.

Reply:
You're right but it looks like it accepted Integer and created the right data type. Since it is already migrated in the testing database I think we should leave it as Integer as long as it doesn't cause any problems in testing it.

Reply:
Since it's just on the testing database (which will be deleted and recreated frequently), I'd prefer we change it.

Reply:
Ill change it then. We do end up dropping this column too

Reply:
If we end up dropping it, what's the point of modifying it? Why bother with this particular migration at all?